docs/workflows/steps/params/README copy.md

```diff
@@ -0,0 +1,103 @@
+# Params
+
+
+
+## What are params?
+
+Params are form inputs that can be added to code steps in a workflow to abstract data from the code and improve reusability. Most actions use params to capture user input (e.g., to allow users to customize the URL, method and payload for the Send HTTP Request action). You can  reference objects in scope (e.g., `event`, `steps`, `process.env`) in params inputs to pass dynamic data to a step.
+
+![](images/image-20200809104652206.png)
+
+  * [Entering Expressions](#entering-expressions)
+    + [Use the object explorer](#use-the-object-explorer)
+    + [Paste a reference from a step export](#paste-a-reference-from-a-step-export)
+    + [Manually enter or edit an expression](#manually-enter-or-edit-an-expression)
+  * [Params Types](#params-types)
+    + [Basic Params](#basic-params)
+    + [Structured Params](#structured-params)
+  * [Manage Params Visibility When Sharing](#sharing-params-values)
+  * [Configuring Custom Params](#configuring-custom-params)
+    + [Adding params to a code step](#adding-params-to-a-code-step)
+    + [Configuring the params form](#configuring-the-params-form)
+
+## Entering Expressions
```

Comment:
It's not immediately clear that I can enter JavaScript expressions, even in the "Manually enter or edit an expression" section - maybe elaborate on that fact?